edgehog-device-runtime-containers/src/store/db/mod.rs

```diff
@@ -0,0 +1,83 @@
+// This file is part of Edgehog.
+//
+// Copyright 2024 - 2025 SECO Mind Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//! Persistent stores of the request issued by Astarte and resources created.
+
+#![allow(dead_code)]
+
+use std::ops::Not;
```

Comment:
Is it still used?
```suggestion
```

Reply:
Yes, it's used for this `.not()` call: https://github.com/edgehog-device-manager/edgehog-device-runtime/blob/7459d3268b5cc3335ea0f8415d7327d5285f111a/edgehog-device-runtime-containers/src/store/db/mod.rs#L57